Content.Server/Shipyard/Systems/ShipyardSystem.Consoles.cs

```diff
@@ -241,16 +230,15 @@ private void OnPurchaseMessage(EntityUid uid, ShipyardConsoleComponent component
         if (TryComp<ShuttleDeedComponent>(targetId, out var deed))
             sellValue = (int) _pricing.AppraiseGrid((EntityUid) (deed?.ShuttleUid!));
 
-        if (ShipyardConsoleUiKey.BlackMarket == (ShipyardConsoleUiKey) args.UiKey || ShipyardConsoleUiKey.Syndicate == (ShipyardConsoleUiKey) args.UiKey) // Unhardcode this please
+        if (component.SalesTax != 0f)
```

Comment:
```suggestion
        if (component.SalesTax > 0.0f)
```
Equality check doesn't play nicely with NaN.  If we need bonuses on selling ships, this needs revision.